sql/schema/core.sql

```diff
@@ -0,0 +1,86 @@
+-- enable UUID of PostgreSQL
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+--users information 
+create table users(
+    id  UUID primary key default gen_random_uuid(),
+    username VARCHAR(255) NOT NULL UNIQUE,
```

Comment:
```suggestion
    username VARCHAR(255) NOT NULL,
```
usenameは基本的に重複許容、uniqueにする特殊な理由があれば教えてください。